src/js/index.js

```diff
@@ -257,7 +257,8 @@ class ReactJsonView extends React.PureComponent {
 
     resetState = () => {
         this.setState({
-            validationFailure: false
+            validationFailure: false,
+            addKeyRequest: false
```

Comment:
this used to work by [setting state directly at render time](https://github.com/mac-s-g/react-json-view/blob/e00a6cf9cc4009dbea0009282dcae6c0623655cd/src/js/index.js#L140) in `index.js`.
it was a hack.  definitely looks better now.

Reply:
awesome!